hack/after-build/update-generated-deep-copies.sh

```diff
@@ -50,13 +50,15 @@ function generate_deep_copies() {
     # Ensure that the version being processed is registered by setting
     # KUBE_API_VERSIONS.
     if [ -z ${ver##*/} ]; then
-        apiVersions=""
+      apiVersions=""
+    else
+      apiVersions="${ver}"
     fi
     KUBE_API_VERSIONS="${apiVersions}" generate_version "${ver}"
   done
 }
 
 # v1 is in the group ""
-DEFAULT_VERSIONS="/ v1 extensions/ extensions/v1beta1 componentconfig/ componentconfig/v1alpha1 metrics/ metrics/v1alpha1"
```

Comment:
Can you document somewhere--here is probably good--what copies are autogenerated by which program, with a hint as to what tasks remain to switch it all over?